scripts/enforce_licence.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
They really don't set python to be any Python now? :cry:

Reply:
It's assumed that simply `python` is Python 2, which usually doesn't exist anymore.

Reply:
Not as per https://peps.python.org/pep-0394/ which suggests python (probably ideally via env) is the only way to do a script which runs on 2 and 3 (which this should). At least in this release and 0.11 I'm rather loath to break compatibility unnecessarily. People doing releases can always either tweak the files or depend on python-is-python3 type packages (although I guess the latter would be naught to force system-wide).

Reply:
When trying either `/usr/bin/python` or `/usr/bin/env python` on a fresh Debian 11, I get:
```
-bash: ./scripts/enforce_licence.py: /usr/bin/python: bad interpreter: No such file or directory
```
or
```
/usr/bin/env: ‘python’: No such file or directory
```
That method is only guaranteed to work in a virtual environment (i.e. venv or virtualenv). The PEP says to use `#!/usr/bin/env python` specifically for virtual environments. The paragraph below says that a compromise must be made outside of a virtual environment.

What is the oldest OS release supported by packagers and/or OLA?

I'll change it back to `#!/usr/bin/env python`, but I think this should really be considered for a breaking change in 0.11. Python 2 has been EOL since January 1, 2020 and isn't included in any active OS distros anymore. I can open a separate issue for that.

Reply:
@peternewman fixed!